lib/reporters/teamcity_reporter.js

```diff
@@ -29,30 +31,41 @@ class TeamcityReporter {
       return;
     }
     this.endTime = new Date();
-    this.out.write('\n\n');
-    this.out.write('##teamcity[testSuiteFinished name=\'testem.suite\' duration=\'' + this._duration() + '\']\n');
+
+    for (var prefix in this.prefixedResults) {
+      if (this.prefixedResults.hasOwnProperty(prefix)) {
+        this.out.write('##teamcity[testSuiteStarted name=\''+prefix+'\']\n');
+        this.prefixedResults[prefix].forEach(this._display.bind(this));
+        this.out.write('##teamcity[testSuiteFinished name=\''+prefix+'\']\n');
```

Comment:
is it intended that `this._duration()` was dropped from the `testSuiteFinished` event?

Reply:
I think previously it was an error as the [jetbrains docs](https://confluence.jetbrains.com/display/TCD65/Build+Script+Interaction+with+TeamCity#BuildScriptInteractionwithTeamCity-ReportingTests) mention `duration` only for *Test start/stop messages*